README.md

```diff
@@ -65,6 +77,82 @@ See [variables.tf] and [examples/] for details and use-cases.
 
 #### Main Resource Configuration
 
+- [**`type`**](#var-type): *(**Required** `string`)*<a name="var-type"></a>
+
+  The type of the notification channel. Valid values are `email`, `slack`, `sms`, `webhook_basicauth` and `pagerduty`.
+
+- [**`project`**](#var-project): *(Optional `string`)*<a name="var-project"></a>
+
+  The ID of the project in which the resource belongs. If it is not provided, the provider project is used.
```

Comment:
```suggestion
  The ID of the project in which the resource belongs. If it is not set, the provider project is used.
```
to avoid *provided* and *provider* in the same phrase.